ResearchKit/Common/ORKQuestionStepViewController.m

```diff
@@ -553,8 +553,6 @@ - (ORKSurveyAnswerCell *)answerCellForTableView:(UITableView *)tableView {
         class = [ORKSurveyAnswerCellForTextField class];
     } else if ([[self.questionStep impliedAnswerFormat] isKindOfClass:[ORKValuePickerAnswerFormat class]]) {
         class = [ORKSurveyAnswerCellForPicker class];
-    } else if ([self.questionStep isFormatVerticalScale]) {
-        class = [ORKSurveyAnswerCellForVerticalScale class];
```

Comment:
No, these lines were removed on purpose as they are only there in order to eventually provide the hardcoded cell height, which is no longer needed.